src/actinia_core/rest/project_management.py

```diff
@@ -120,14 +122,17 @@ def get(self):
                     if os.path.isdir(mapset_path) and os.access(
                         mapset_path, os.R_OK & os.X_OK
                     ):
-                        locations.append(dir)
-        if locations:
+                        projects.append(dir)
+        if projects:
+            param = {"status": "success"}
+            grass_version_s = G_VERSION["version"]
+            grass_version = [int(item) for item in grass_version_s.split(".")[:2]]
+            if grass_version >= [8, 4]:
+                param["projects"] = projects
```

Comment:
This might break with `LocationListResponseModel`

Reply:
Fine if this PR stays open as long as it takes to also adjust actinia-api but I would suggest to do adjustments in a separate PR.